docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   db:
-    image: postgres:10-alpine
+    image: postgres:13-alpine
```

Comment:
Why not use the latest stable? `postgres:15-alpine`

Reply:
I only tried with the same version that is for github actions, I'm happy using the last version of postgres for locally tests.  I just updated to `postgres:15-alpine`.

Reply:
I tested it locally also.

Reply:
We were using `postgres:13` just because is the last version supported by AWS, but `postgres:15` should do it